metronome/networking/src/io/iohk/metronome/networking/NetworkEvent.scala

```diff
@@ -0,0 +1,28 @@
+package io.iohk.metronome.networking
+
+import java.net.InetSocketAddress
+
+/** Events we want to trace. */
+sealed trait NetworkEvent[K]
```

Comment:
Maybe another useful ones would be: `MessageReceived` and `ConnectionError` ?

Reply:
Is `ConnectionError` different from `ConnectionFailed`?

Sure, I'll add `MessageReceived` and `MessageSent` as well.

Reply:
`ConnectionFailed` is when we call to remote peer and fail,  and `ConnectionError` would be when we are listening for incoming event from peer and receive lets say `DecodingError`

Reply:
Done, added `ConnectionReceiveError`, `ConnectionSendError`, `ConnectionUnknown`, `MessageSent` and `MessageReceived`.